### PR TITLE
Drop README in favour of README.md

### PR DESCRIPTION
**Pull request by lfaraone**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -29,9 +29,8 @@ def get_package_data(package, base_dir):
     "package_data": package_data,
     "scripts": ["bin/grouper-api", "bin/grouper-fe", "bin/grouper-ctl"],
     "description": "Self-service Nested Group Management Server.",
-    # PyPi, despite not parsing markdown, will prefer the README.md to the
-    # standard README. Explicitly read it here.
-    "long_description": open("README").read(),
+    # TODO(lfaraone): Check whether this is still needed for PyPI.
```
Comment (lfaraone) [inline setup.py]: If we don't specify a `long_description`, will PyPI DTRT and use the `README.rst`?

Comment (gmjosack) [inline setup.py]: I believe that it will, though I'm not completely positive.

**Comment by gmjosack** [comment]
Can you kill the Makefile now too?

**Comment by lfaraone** [comment]
Sure, will do.

**Pull request merged**